File: Runtime/RuntimeApplyScene/Classes/BaseViewController.m
//
//  BaseViewController.m
//  RuntimeApplyScene
//
//  Created by 朱献国 on 2020/11/2.
//  Copyright © 2020 朱献国. All rights reserved.
//

#import "BaseViewController.h"


@interface BaseViewController ()

@end

@implementation BaseViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    [self totalInit];
}

- (void)totalInit {
    self.navigationItem.title = @"Runtime";
    self.view.backgroundColor = UIColorRandomColor;
}

@end

File: Swift/U17_Demo/Classes/Base/BaseViewController.swift
//
//  BaseViewController.swift
//  U17_Demo
//
//  Created by 朱献国 on 2019/8/1.
//  Copyright © 2019 朱献国. All rights reserved.
//

import UIKit

class BaseViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()

        view.backgroundColor = UIColor.gray
    
    }

}


extension BaseViewController {
    override var preferredStatusBarStyle: UIStatusBarStyle {
        return .default
    }
}

